bitnami/drupal/templates/deployment.yaml

```diff
@@ -56,7 +56,10 @@ spec:
       hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.hostAliases "context" $) | nindent 8 }}
       # yamllint enable rule:indentation
       {{- end }}
-      initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+      initContainers:
+        {{- if .Values.initContainers }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
```

Comment:
If the indent is at 8 blocks, but we are showing it at 10 blocks, doesn't it make it a bit inconsistent?

Reply:
I saw examples using both ways, but yes, it is more consistent the opposite